wee_slack.py

```diff
@@ -1624,7 +1624,7 @@ def open(self, update_remote=True):
             self.eventrouter.receive(s)
         if update_remote:
             if "join" in SLACK_API_TRANSLATOR[self.type]:
-                s = SlackRequest(self.team.token, SLACK_API_TRANSLATOR[self.type]["join"], {"user": self.user}, team_hash=self.team.team_hash, channel_identifier=self.identifier)
+                s = SlackRequest(self.team.token, SLACK_API_TRANSLATOR[self.type]["join"], {"user": self.user, "return_im": "true"}, team_hash=self.team.team_hash, channel_identifier=self.identifier)
```

Comment:
The type of "return_im" is specified to be boolean, not string, so this should be `True`.

Reply:
Good call, I saw `"return_im": "true"` on line 1289 and just did same for consistency here (identical requests), but I'll update both to use `True`.

Reply:
Ah, I hadn't noticed that line. It seems weird that this is done in both `create_buffer` and `open` though, especially since `create_buffer` is called from `open`, so it will send two requests when you open a DM that doesn't have an existing buffer.

But this is existing weirdness, and your PR is an improvement either way, so I'll merge it.